vendor/BUILD

```diff
@@ -393,8 +393,10 @@ filegroup(
         "//vendor/golang.org/x/lint:all-srcs",
         "//vendor/golang.org/x/net/context:all-srcs",
         "//vendor/golang.org/x/net/html:all-srcs",
+        "//vendor/golang.org/x/net/http/httpguts:all-srcs",
         "//vendor/golang.org/x/net/http2:all-srcs",
         "//vendor/golang.org/x/net/idna:all-srcs",
+        "//vendor/golang.org/x/net/internal/socks:all-srcs",
         "//vendor/golang.org/x/net/internal/timeseries:all-srcs",
         "//vendor/golang.org/x/net/lex/httplex:all-srcs",
```

Comment:
This is probably why the godeps test didn't pass: https://gubernator.k8s.io/build/kubernetes-jenkins/pr-logs/pull/71501/pull-kubernetes-godeps/1086